sdk/src/app/app.component.html

```diff
@@ -1,5 +1,10 @@
 <div className="App">
   <h1>Hi! This is an ImageKit Angular SDK Demo!</h1>
+  <ik-video src="https://ik.imagekit.io/demo/sample-video.mp4"></ik-video>
+  <p></p>
+  <ik-video src="https://ik.imagekit.io/demo/sample-video.mp4" [transformation]="transformationOne"></ik-video>
+  <p></p>
+  <ik-video src="https://ik.imagekit.io/demo/sample-video.mp4" [transformation]="transformationTwo"></ik-video>
```

Comment:
I think it is an object array, not a string. Maybe you can use an example from react?

Reply:
transformation